infrastructure/cdn-in-a-box/traffic_ops/config.sh

```diff
@@ -32,9 +32,14 @@
 # TO_HOST
 # TO_PORT
 # TP_HOST
+# TV_DB_USER
+# TV_DB_NAME
+# TV_DB_SERVER
+# TV_DB_USER_PASS
+# TV_DB_PORT
```

Comment:
Can these be alphabetized? Some of the rest of the list is also unsorted, might as well sort it all.